source/guides/tooling/code-coverage.md

```diff
@@ -121,6 +121,8 @@ Cypress does not instrument your code - you need to do it yourself. The golden s
 - Using the {% url "nyc" https://github.com/istanbuljs/nyc %} module - a command-line interface for the {% url "Istanbul" https://istanbul.js.org %} library
 - As part of your code transpilation pipeline using the {% url "`babel-plugin-istanbul`" https://github.com/istanbuljs/babel-plugin-istanbul %} tool.
 
+## Instrumenting code: Using NYC
```

Comment:
I would not repeat "Instrumenting code:" here again - because it is already the higher level heading

Reply:
Sure - this makes sense